source/Sashimi.Terraform.Tests/ActionHandlersFixture.cs

```diff
@@ -363,17 +368,25 @@ void PopulateVariables(TestActionHandlerContext<Program> _)
             output = ExecuteAndReturnResult(typeof(TerraformApplyActionHandler), PopulateVariables, "Azure");
             output.OutputVariables.ContainsKey("TerraformValueOutputs[url]").Should().BeTrue();
             output.OutputVariables["TerraformValueOutputs[url]"].Value.Should().Be(expectedHostName);
-            await MakeRequest();
+            await AssertRequestResponse(HttpStatusCode.Forbidden);
 
             ExecuteAndReturnResult(typeof(TerraformDestroyActionHandler), PopulateVariables, "Azure");
-            Func<Task> request = async () => await MakeRequest();
-            request.Should().Throw<HttpRequestException>().WithMessage("No such host is known.");
 
-            async Task MakeRequest()
+            //This will throw on some platforms and return "NotFound" on others
```

Comment:
That is weird, so it fails resolving dns?

Reply:
Yes since the host isn't there anymore right